### PR TITLE
make bparees an api approver

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -9,4 +9,6 @@ approvers:
   - jwforres
   - knobunc
   - sjenning
-  - soltysh
\ No newline at end of file
+  - soltysh
+  - bparees
+  - sdodson
```
Comment (bparees) [inline OWNERS]: @sdodson fyi

**Comment by bparees** [comment]
@eparis can you approve? my understanding is all group leads are supposed to have this.

File: OWNERS (hunk 1 above)
Comment (sdodson) [reply OWNERS]: I'd be totally fine if you omit me, at least for the next 2 months.

Comment (bparees) [reply OWNERS]: heh, fair enough. updated.

**Pull request merged**